.gitignore

```diff
@@ -94,3 +94,5 @@ doxy/
 
 # aspell backups
 *.bak
+
+tests/unit/C/*/*/*/build/*
```

Comment:
Now it has conflicts regarding this matter, and it would be advisable to test specific `.gitignore` file in the C-based tests directory.

Also, unless you have a very specific reason to target only three levels deep to ignore, we might need to use with wildcard pattern:

```
tests/unit/C/**/build/
tests/unit/C/**/build/*
```
Ignores all build/ directories recursively under `tests/unit/C/`. It also Ignores both the directories and their contents